### DISCUSSION TITLE
Action that opens a PR not trigger another workflow

### DISCUSSION THREAD

**Discussion by jaxxstorm** [community/community#25379] [Actions]
<p>I have a GitHub action that opens a pull request when there are commits to a specific branch:</p>
<pre><code class="lang-auto">env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
jobs:
  open-pull-request:
    name: open pull request
    runs-on: ubuntu-latest
    steps:
    - name: Checkout Repo
      uses: actions/checkout@v2
    - name: Create Pull Request
      uses: repo-sync/pull-request@v2
      with:
        github_token: ${{ secrets.GITHUB_TOKEN }}
name: pr-automation
"on":
  push:
    branches:
    -automation
</code></pre>
<p>And I also have a workflow that runs with event <code>{ on: pull-request { branches: master } } }</code></p>
<p>When the there’s a commit to the <code>automation</code> branch, I’d expect 2 workflows to run:</p>
<ul>
<li>The automation workflow which opens a pull request</li>
<li>the pull request workflow, because a new PR has been opened against master</li>
</ul>
<p>The second workflow isn’t currently running. I suspect it may be related to the GITHUB_TOKEN permissions, but I can’t be sure. Any pointers?</p>

**Comment by vaadin-miki** [comment, marked as answer]
<p>Unless the token is your personal access token, actions will not trigger other acctions - to prevent endless loops.</p>
<p>To remedy that, simply create a personal access token, store it as a secret and use that secret instead.</p>

**Comment by jaxxstorm** [comment]
<p>I did try that, by passing the PAT to the “open pull request” job. It didn’t seem to rectify the issue.</p>

**Comment by vaadin-miki** [comment]
<p>Just asking, isn’t that a syntax error: <code>-automation</code> - shouldn’t it be <code>- automation</code> (with a space)?</p>

**Comment by jaxxstorm** [comment]
<p>yes, it was a copy/paste fail - the original workflow seems fine</p>
